packages/backend/src/controllers/projectController.ts

```diff
@@ -749,10 +749,18 @@ export class ProjectController extends BaseController {
     @SuccessResponse('204', 'Deleted')
     @Delete('{projectUuid}/tags/{tagUuid}')
     @OperationId('deleteTag')
-    async deleteTag(@Path() tagUuid: string, @Request() req: express.Request) {
+    async deleteTag(
+        @Path() tagUuid: string,
+        @Request() req: express.Request,
+    ): Promise<ApiSuccessEmpty> {
         await this.services.getProjectService().deleteTag(req.user!, tagUuid);
 
-        this.setStatus(204);
+        this.setStatus(200);
```

Comment:
if I have 204 status it doesn't set the endpoint call as successful when called through the react-query hook 😢 Since I couldn't find a quick solution I just copied what is being done for other DELETE endpoints @almeidabbm